templates/api.html

```diff
@@ -66,6 +66,7 @@ <h3>
 repository
 autoupdate
 author
+sri
```

Comment:
SRI implementation here is under "assets", so adding it here may not be correct.